SConstruct

```diff
@@ -470,12 +470,18 @@ for env in [test_env, client_env, env]:
 
     env.Append(CPPDEFINES = ["HAVE_CONFIG_H"])
 
+    if "clang" in env["CXX"]:
+# Silence warnings about unused -I options and unknown warning switches.
+        env.AppendUnique(CCFLAGS = Split("-Qunused-arguments -Wno-unknown-warning-option -Werror=non-virtual-dtor"))
```

Comment:
Likewise, it's strange that `-Werror=non-virtual-dtor` is only enabled by default with Clang.